src/features/encoders/avif/client/index.tsx

```diff
@@ -67,18 +75,18 @@ export class Options extends Component<Props, State> {
     return {
       options,
       lossless,
-      quality: maxQuant - cqLevel,
+      quality: cqLevel,
       separateAlpha,
-      alphaQuality:
-        maxQuant -
-        (separateAlpha ? options.cqAlphaLevel : defaultOptions.cqLevel),
+      alphaQuality: separateAlpha
+        ? options.cqAlphaLevel
+        : defaultOptions.cqLevel,
```

Comment:
Change lines 81-82 from

```
        ? options.cqAlphaLevel
        : defaultOptions.cqLevel,
```

to


```
        ? options.qualityAlpha
        : defaultOptions.quality,
```

Note: I don't know the distinction between `options` and `defaultOptions`.

Reply:
`options` is the user-selected codec parameters, `defaultOptions` is just the default parameters for the codec when it is selected for the first time. I'll change the field names and make sure they match correctly.

Reply:
Thanks for the explanation. If `separateAlpha` is false, why do we use `defaultOptions.cqLevel` instead of `options.cqLevel`? The latter seems more reasonable to me.

Reply:
Yep sounds good to me too, I'll make the changes